carma_nav2_port_drayage_demo/src/port_drayage_demo.cpp

```diff
@@ -16,10 +16,28 @@
 
 #include <nav2_msgs/action/follow_waypoints.hpp>
 #include <nav2_util/lifecycle_node.hpp>
-#include <nlohmann/json.hpp>
 
 namespace carma_nav2_port_drayage_demo
 {
+
```

Comment:
Could you add documentation on the purpose of `operationToString()` and why it's commented out